microservices/nris_api/backend/app/nris/resources/document.py

```diff
@@ -0,0 +1,34 @@
+from flask import request
+from flask_restplus import Resource
+
+from werkzeug.exceptions import BadRequest, NotFound, InternalServerError
+
+from app.extensions import api, db
+from app.nris.utils.access_decorators import requires_role_nris_view
+
+from app.nris.models.document import Document as Model, DOCUMENT_RESPONSE_MODEL as RESPONSE_MODEL
+
+module_path = 'documents'
+filter_fields = ['document_type', 'file_name']
```

Comment:
Any reason this isn't scoped to the class and is global?

Reply:
Primarily to group "configuration", although I could argue that even though today only the List resources use the filter_fields, both resources should be able to use them.